lib/srv/regular/sshserver_unix_test.go

```diff
@@ -63,29 +62,18 @@ func TestValidateListenerSocket(t *testing.T) {
 		return left, listenerFD
 	}
 
-	u, err := user.Current()
-	require.NoError(t, err)
-
 	tests := []struct {
```

Comment:
Does the existing test suite already reproduces the bug fixed here?

If not, could we make sure we have test coverage for it (and that it fails without the fix)?